js/collections/UsersShort.js

```diff
@@ -0,0 +1,31 @@
+/* use this collection for any list of users that can be fetched as an array of guids */
+
+import { Collection } from 'backbone';
+import UserShort from '../models/UserShort';
+import app from '../app';
+
+
+module.exports = Collection.extend({
+  /* we have to use the older style for this collection, the ES6 style creates a bug where models
+  cannot be removed using their ids */
+
+  initialize: function (models, options) {  // eslint-disable-line object-shorthand
```

Comment:
I think you could use function short-hand here (and therefore get rid of the lint override). I think it was the ES6 class behavior that was messing up the collection. I don't think using function short-hand on the object being passed to 'extend' would cause problems.